start.sh

```diff
@@ -8,4 +8,4 @@ fi
 source .venv/bin/activate
 python -m pip install uv
 uv sync
-export EADK_DISCORD_CONFIG_PATH=.bot_config.toml && uv run eadk_discord
+export uv run eadk_discord .bot_config.toml
```

Comment:
why do you still have `export`??

Reply:
yeah, I realised this when I actually ran it on the server. It's fixed on the server